src/comm/USBBoardInfo.json

```diff
@@ -54,6 +55,8 @@
         { "regExp": "^PX4 BL FMU v6U.x$",   "boardClass": "Pixhawk" },
         { "regExp": "^PX4 FMU v6X.x$",      "boardClass": "Pixhawk" },
         { "regExp": "^PX4 BL FMU v6X.x$",   "boardClass": "Pixhawk" },
+        { "regExp": "^PX4 FMU v5X.x$",      "boardClass": "Pixhawk" },
+        { "regExp": "^PX4 BL FMU v5X.x$",   "boardClass": "Pixhawk" },
         { "regExp": "^PX4 FMU v5.x$",       "boardClass": "Pixhawk" },
```

Comment:
@dagar not sure how is the regex working on the other end, but would the dot "." on this line need to be escaped to avoid the regex from catching an X or any other char other than the dot?